org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/RenameService2.xtend

```diff
@@ -77,31 +114,137 @@ class RenameService2 implements IRenameService2 {
 			} else {
 				issueAcceptor.add(FATAL, 'Loaded resource is not an XtextResource', context.resource.URI)
 			}
+			issueAcceptor.checkSeverity
 			return workspaceEdit
+		].exceptionally [ exception |
+			val rootCause = Throwables.getRootCause(exception);
+			if (rootCause instanceof FileNotFoundException) {
+				if (shouldPrepareRename) {
+					return null
```

Comment:
Report issue?

Reply:
I did not want to. It is already traced, one can glance into the logs. From the LSP's perspective, it does not add any value if we "reporting" the issues.